conf/st2.conf.sample

```diff
@@ -29,7 +29,7 @@ max_page_size = 100
 # True to mask secrets in the API responses
 mask_secrets = True
 # StackStorm API server host
-host = 127.0.0.1
```

Comment:
Pushed some changes - 9784bde065f43c8e210d3164e1be94847aa78688, ea66b86fabcf1898f09d59ecf3d6d288d8bf8f87, 971582612a1ad234fb9e95387db26a07d599d7f9.

Will see if e2e tests pass, then we should be fine.